### DISCUSSION TITLE
Feature Request: Show charge locations on map in Trip dashboard

### DISCUSSION THREAD

**Discussion by khorton** [teslamate-org/teslamate#3415] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The Trip dashboard has a nice map showing the drives, and a list of charges.  But, the map does not show the charges

### Expected Behavior

Add icons on the map to show the locations of the charges.  For extra points, have different icons for DC charging and AC charging.  For full marks, allow the user to provide custom icons if he/she desires and also allow the charge icons on the map to be enabled or disabled using the Settings.

### Steps To Reproduce

Display Trip dashboard

### Relevant log output

```shell
None
```

### Type of installation

Docker

### Version

v1.27.3

**Comment by Dulanic** [comment]
PR's are accepted. Converting to discussion for now to help cleanup issues.

**Comment by jaypark0006** [comment]
I’m also very interested in this. 

For example, if I were to start a journey that would take about 1 week, I’d want to have a map that shows the route I traveled, where I charged, where I rested, etc. 

I need to think about how this could be achieved.
